tasklist/common/src/main/java/io/camunda/tasklist/entities/listview/ListViewJoinRelation.java

```diff
@@ -23,6 +23,11 @@ public ListViewJoinRelation(final String name) {
     this.name = name;
   }
 
+  public ListViewJoinRelation(final String name, final String parent) {
+    this.name = name;
+    this.parent = Long.valueOf(parent);
```

Comment:
The nullity check wont be no longer necessary, for this situation, because this constructor just will be called when parent exists (flowNode/scopeKey) 

For the cases that parent is null (process), we have another constructor
```
  public ListViewJoinRelation(final String name) {
    this.name = name;
  }
```